py_code.py

```
@@ -0,0 +1,776 @@
+                                                                message='功能点生成异常')
+                logger.error(f'{self.logger_key}, 功能点获取异常. 耗时: {calculate_time_diff(st, et)}')
+                raise Exception(f'{self.logger_key}, 功能点获取异常. data: {dict(content=results)}')
+
+            # 选取最多功能点为结果
+            func_point_ai_res = max(results, key=lambda x: x.count('**** 功能点'))
+            self.func_point_mindmap = func_point_ai_res
+        logger.info(f'{self.logger_key}, 功能点获取完成. 总耗时: {calculate_time_diff(st, et)}')
+        return self.func_point_mindmap
+
+def write_subnet(account_id, vpc_vid, cidr_pos):
+    row = subnet_row(account_id, cidr_pos)
```

Ruff (0.8.2)

Undefined name `subnet_row`

(F821)